### ISSUE TITLE
Divide error LLS stage 4

### ISSUE THREAD

**Issue by gooeyPhantasm** [spaztron64/th98tuc_site#11]
![image](https://user-images.githubusercontent.com/89147140/145922404-30079094-55f7-491a-a915-d9a0e6f38436.png)
This error seems to occasionally occur when fighting against Marisa. The list of known issues mentions this as a possibility against Kurumi but not Marisa.

**Comment by spaztron64** [comment]
Yeah, this has been brought to my attention lately. Will make sure to properly document it in the next release.
I can't make any guarantees for a proper fix unfortunately, as we don't yet properly understand why this crash happens in the first place.

**Comment by nmlgc** [comment]
Both Kurumi's and Marisa's Divide Error crashes are now researched and documented, and fixes are available as well:

https://rec98.nmlgc.net/blog/2022-04-18
